### DISCUSSION TITLE
c `#include <>` instead of `#include ""`

### DISCUSSION THREAD

**Discussion by duarm** [neovim/neovim#26433] [Q&A] [answered]
hello, I can't find a way to opt into including with <> instead of "", whenever lsp automatically includes something for me, it always uses "" by default, so I get something like this

![image](https://github.com/neovim/neovim/assets/26150581/14e6e98e-ce4b-49f5-8371-c5adcf6a997e)

**Comment by izza-mus** [comment]
Hey, did you find a solution to this problem?

**Reply by clason** [reply]
This is not a Neovim issue; check whatever language server you are using.

**Reply by duarm** [reply, marked as answer]
Here's the tracking issue on clangd
https://github.com/clangd/clangd/issues/1247

not sure if there's a way to hook into the lsp message and change "" to <> as a workaround, it's not that much of a problem so I didn't look further into it after asking for help
